utils/events.py

```diff
@@ -6,7 +6,18 @@
 
```

Comment:
Just some examples of how it works, what's required for setup, how you access events etc